src/fastmeteo/web/client.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+import httpx
+import pandas as pd
+
+
+class Client:
+    def __init__(
+        self,
+        server: str = "127.0.0.1",
+        port: int = 9800,
+        client: httpx.Client | None = None,
+    ) -> None:
+        self.server = server
+        self.port = port
+        self.client = client or httpx.Client()
```

Comment:
It's common to allow people to define their client, as they want to set some specific config/header/whatever to match their specific needs.
In practice, it also allows to use a `TestClient` for the `pytest` part.